stored_requests/backends/db_fetcher/fetcher_test.go

```diff
@@ -86,6 +125,20 @@ func TestEmptyResponse(t *testing.T) {
 	assertMapLength(t, 0, storedImps)
 }
 
+func TestEmptyResponseFetchResponses(t *testing.T) {
+	mockQuery := "SELECT id, data, dataType FROM my_table WHERE id IN (?, ?)"
+	mockReturn := sqlmock.NewRows([]string{"id", "data", "dataType"})
+
+	mock, fetcher := newFetcher(t, mockReturn, mockQuery, "stored-resp-id", "stored-resp-id-2")
+	defer fetcher.db.Close()
+
+	storedResponses, errs := fetcher.FetchResponses(context.Background(), []string{"stored-resp-id", "stored-resp-id-2"})
+
+	assertMockExpectations(t, mock)
+	assertErrorCount(t, 0, errs)
+	assertMapLength(t, 0, storedResponses)
+}
+
```

Comment:
Similar comment to above, where I think the TestGoodRepsonseFetchResponses, TestPartialResponse, and TestEmptyResponse tests could be consolidated into one table driven test.

Reply:
Done